test/public/envs/overview.test.js

```diff
@@ -171,15 +173,15 @@ module.exports = () => {
             // eslint-disable-next-line no-undef
             model.envs.overviewModel.pagination.itemsPerPage = 1;
         });
-        await waitForTimeout(100);
+        await waitForTableLength(page, 1);
 
         // Expect the page five button to now be visible, but no more than that
         await page.waitForSelector('#page5');
         await page.waitForSelector('#page6', { hidden: true });
 
         // Expect the page one button to have fallen away when clicking on page five button
         await pressElement(page, '#page5');
-        await waitForTimeout(100);
+        await page.waitForSelector('#page1', { hidden: true });
         const pageOneButton = await page.$('#page1');
         expect(Boolean(pageOneButton)).to.be.false;
```

Comment:
Simply waiting for it to be hidden is enough

Reply:
Done